net.purrdata.PurrData.yml

```diff
@@ -10,8 +10,9 @@ finish-args:
 - --socket=x11
 - --socket=wayland
 - --device=dri
-# Audio access
-- --socket=pulseaudio
```

Comment:
please leave these. They are still needed.

Reply:
Thanks! I am copying from the Guitarix flatpak, and missed that it still opened this socket

Reply:
Guitarix require JACK that's the big difference. (and problem)